templates_jinja2/match_details.html

```diff
@@ -53,7 +53,7 @@
       {% endif %}
       <p>
         <a class="btn btn-success" href="/suggest/match/video?match_key={{match.key_name}}" target="_blank"><span class="glyphicon glyphicon-plus"></span> Add videos</a>
-        <a class="btn btn-primary" href="https://www.youtube.com/results?search_query={{match.verbose_name|urlencode}}+{{event.year}}+{{event.name|urlencode}}" target="_blank"><span class="glyphicon glyphicon-search"></span> Search YouTube</a>
+        <a class="btn btn-primary" href="https://www.youtube.com/results?search_query={{match.key_name}}" target="_blank"><span class="glyphicon glyphicon-search"></span> Search YouTube</a>
```

Comment:
The current format is `<match type>(<set>)M<match num> <event name>`, see [this example](https://www.youtube.com/user/NewEnglandFIRST/playlists)

You can build that with `{{match.short_name|urlencode}}+{{event.name|urlencode}}`

See the definition of [match.short_name](https://github.com/the-blue-alliance/the-blue-alliance/blob/master/models/match.py#L176)